fastly/resource_fastly_service_v1.go

```diff
@@ -1574,6 +1575,7 @@ func resourceServiceV1Update(d *schema.ResourceData, meta interface{}) error {
 		"splunk",
 		"blobstoragelogging",
 		"httpslogging",
+		"logging_ftp",
```

Comment:
@ezkl should we stick with the pattern of the `logging` suffix (i.e., `ftplogging`) rather than changing to use `logging_` as a prefix, just for consistency with what already exists?

This would be the only necessary change that I see worth going back and applying.

cc @phamann

Reply:
I agree with this decision of starting to introduce consistency with `logging_foo`